src/types/Schedule.ts

```diff
@@ -0,0 +1,56 @@
+import { arg, extendType, inputObjectType, intArg, nonNull, objectType } from 'nexus';
+import { CourseSection } from './Course/Section';
+import { Date } from './Date';
+import { Response } from './Response';
+import { Term } from './Term';
+
+export const GenerateScheduleInput = inputObjectType({
```

Comment:
soo just so I understand it better, if frontend team calls this endpoint and sends a term, **backend** will call the algorithm to generate schedule?

Reply:
Most likely, yes. I imagine algorithm will just expose an API endpoint or something.